### ISSUE TITLE
Collection wide royalties

### ISSUE THREAD

**Issue by GummyLabs** [glow-xyz/nftoken#146]
Please consider the addition of collection wide royalties that can be optionally be overriden by specific NFT royalty accounts.

This would be huge.

**Comment by vpontis** [comment]
Thanks for that suggestion! Yea, definitely considering that 👍. 

What's the motivation for this request? It's expensive to set up a `nft_creators` account for every NFT?

**Comment by GummyLabs** [comment]
Yes that is correct for cost reasons. It's also very common to only want to set a single royalty setting across the collection.
It's also far easier to edit the royalties across the collection later rather then having to send 1000s of update instructions.
